OneCommunityGlobal/HighestGoodNetworkApp
# updated unit testing for textsearchbox

# Description
Added unit tests for textSearchBox

## How to test:
1. check into current branch
2. do `npm install` and `npm test TextSearchBox.test.js` to run this PR locally
4. verify all 6 tests passed

## Screenshot:
<img width="752" alt="Screenshot 2023-12-07 at 11 00 29 AM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104778270/c65987ac-55fd-4296-9dd1-30c1f6532277">
